nservicebus/upgrades/5to6.md

```diff
@@ -98,4 +98,26 @@ NServiceBus will automatically make correlated saga properties unique without th
 
 ## MSMQ ReturnToSourceQueue.exe
 
-The MSMQ ReturnToSourceQueue.exe tool is now deprecated. The code for this tool has been moved to [ParticularLabs/MsmqReturnToSourceQueue](https://github.com/ParticularLabs/MsmqReturnToSourceQueue) repository. See the readme in that repository for full details.
\ No newline at end of file
+The MSMQ ReturnToSourceQueue.exe tool is now deprecated. The code for this tool has been moved to [ParticularLabs/MsmqReturnToSourceQueue](https://github.com/ParticularLabs/MsmqReturnToSourceQueue) repository. See the readme in that repository for full details.
+
+## Message handler ordering
+In Version 6 we simplified the message handler ordering APIs drastically. Previously there were multiple ways to specify the handler ordering:
```

Comment:
kill off `drastically`. either the snippets show this, in which case the word is redundant, or the reader does not think the snippets show this, in which case they thing we are exaggerating.